tools/file_packager.py

```diff
@@ -712,6 +710,17 @@ def was_seen(name):
 
     ret += r'''
       function fetchRemotePackage(packageName, packageSize, callback, errback) {
+        if (typeof process === 'object') {
+          fs = require('fs');
```

Comment:
@sbc100 Maybe

```suggestion
          const fs = require('fs');
```
? I'm getting a `ReferenceError: fs is not defined` otherwise.

Reply:
Nevermind, I see it was fixed in https://github.com/emscripten-core/emscripten/pull/14372